packages/grafana-toolkit/package.json

```diff
@@ -30,85 +30,19 @@
     "LICENSE_APACHE2"
   ],
   "scripts": {
-    "build": "yarn clean && tsc -p tsconfig.json && yarn copy-files",
-    "copy-files": "node ./scripts/copy-files.js",
-    "clean": "rimraf ./dist ./compiled ./sass ./package.tgz",
+    "build": "yarn clean && tsc -p tsconfig.json",
+    "clean": "rm -rf ./dist ./compiled ./sass ./package.tgz",
     "prepack": "mv ./src ./src_bak && cp -r ./dist/src ./src",
-    "postpack": "rimraf ./src && mv ./src_bak ./src",
+    "postpack": "rm -rf ./src && mv ./src_bak ./src",
```

Comment:
does this have any implications for windows machines? believe that's the whole reason for including `rimraf`. or do we not care since it's now removed? 🤔

Reply:
The scripts are already using `mv` (see prepack) which doesn't work on windows and we support windows under WSL https://grafana.com/blog/2021/03/03/how-to-set-up-a-grafana-development-environment-on-a-windows-pc-using-wsl/